node/src/node_configurator/node_configurator_standard.rs

```diff
@@ -640,15 +666,25 @@ mod tests {
     }
 
     #[test]
-    fn configure_cryptdes_handles_missing_password_with_uninitialized_cryptdes() {
```

Comment:
Each of these tests has multiplied into three tests: one with `--new-public-key off`, one with `--new-public-key on`, and one with `--new-public-key` completely missing.